cmd/vic-machine/create/create.go

```diff
@@ -181,6 +182,12 @@ func (c *Create) Flags() []cli.Flag {
 		},
 
 		// volume
+		cli.StringFlag{
+			Name:        "default-volume-store, vsd",
```

Comment:
Do we need a flag for `default`?  I though we were going to complain if a `volume-store` with the name `default` wasn't specified unless a flag was added to explicitly not created on.  Like `--no-default-volumestore` or some such.  This overloads an already existing flag of `volume -store` so the question is what if someone uses that flag and specifies a name of "default" using it.  Isn't that the same thing?